auth-web/package.json

```diff
@@ -21,7 +21,7 @@
     "moment": "^2.24.0",
     "regenerator-runtime": "^0.13.3",
     "register-service-worker": "^1.6.2",
-    "sbc-common-components": "^2.3.1",
+    "sbc-common-components": "^2.3.4",
```

Comment:
u will have to check in the package-lock as well..

Reply:
Added package-lock